File: OOPS/customexception-project/src/com/swbhavtechlabs/myexception/Even.java
package com.swbhavtechlabs.myexception;

public class Even {

	private int number;
	public boolean isEven(int number)
	{
		if(number%2 ==0)
		{
			//try
			//{	
				throw new NotEvenException();
			//}
			//catch(Exception e)
			//{
			//	e.printStackTrace();
			//	return true;
			//}
			
		}
		return false;
		
	}
}

File: CSharpOOPS/customexception/customexception/myexception/Even.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace customexception.myexception
{
    class Even
    {
        private int _number;

        public int Number
        {
            get
            {
                return _number;
            }
            set
            {
                _number = value;
            }
        }

        public bool isEven()
        {
            if(_number % 2 !=0)
            {
                throw new myexception.NotEvenException(" Not a Even Number");
            }
            return true;
        }
    }
}
